tensorboard/webapp/metrics/views/card_renderer/image_card_container.ts

```diff
@@ -284,8 +286,30 @@ export class ImageCardContainer implements CardRenderer, OnInit, OnDestroy {
       this.stepValues$
     ).pipe(
       map(([stepIndex, selectedTime, selectedSteps, stepValues]) => {
-        if (!selectedTime || selectedTime.clipped || selectedSteps.length === 0)
-          return stepIndex;
+        if (!selectedTime || selectedTime.clipped) return stepIndex;
+
+        // When there is no selected steps. We check if start step is
+        // "close" enough to a step value and move it.
+        if (selectedSteps.length === 0) {
+          if (stepValues.length === 1) return stepIndex;
+
+          const step = selectedTime.startStep;
+          for (let i = 0; i < stepValues.length - 2; i++) {
+            const currentSelectedStep = stepValues[i];
```

Comment:
Maybe a little clearer to rename these to `currentStepValue` and `nextStepValue`.

(It might also be worth incorporating `value` into the currently-named `step` variable -- but I'm not certain how best to incorporate that - `stepValue` still seems insufficient)